uitwerking issue-32 (#40)
pldn/nederlands-profiel-voor-stelselcatalogi: respec/Inleiding.md

Voor het beschrijven van het profiel maken we gebruik van bestaande W3C standaarden. Dit leidt tot een abstracte syntax van het profiel uitgedrukt in de SHACL standaard. Daarnaast geven we een taalbinding van deze abstracte syntax naar andere bestaande W3C standaarden, zoals SKOS en DublinCore. Voor de abstracte syntax gebruiken we URI's die uitgedrukt worden in de Engelse taal, om zo direct een internationale aansluiting te houden. De abstracte syntax zal daarnaast Nederlandse labels kennen die in de deze lopende (Nederlandstalige) tekst zal worden gebruikt. De meerwaarde van het gebruik van SHACL is dat deze praktische handvatten biedt bij het toepassen van deze bestaande standaarden, inclusief een validatiemogelijkheid. Door het geven van de taalbinding naar de andere W3C standaarden bieden we een profiel dat direct kan worden begrepen door de internationale (internet) gemeenschap. Specifiek voor de Nederlandse (overheids)situatie geldt dat we hierbij aansluiten op de standaarden zoals opgenomen op de Pas-Toe-Of-Leg-Uit lijst conform de Nederlandse Overheids Referentie Architectuur (NORA).

### Algemene overeengekomen richtlijnen

> Om het profiel succesvol te laten zijn dient het breed gedragen te zijn. Om dit te realiseren hebben we een breed scala aan vertegenwoordigers samen gesteld vanuit diverse (semi-overheid)instanties. Vanuit deze werkgroep hebben we een aantal richtlijnen opgesteld voor het profiel. Deze lijst is niet uitputtend en volledig, maar dient als een basis.

Reviewer: Ik las "deze lijst" als deze lijst van vertegenwoordigers. Beter: "onderstaande lijst".
Of misschien nog beter: een aparte sectie over de samenstelling van de werkgroep en de gevolgde werkwijze. Hoeft niet heel formeel, maar er mag best iets meer toegelicht worden hoe het profiel toe stand is gekomen.

Author: klopt, wat ambigu. Ik heb de ambiguïteit nu opgelost door de term "lijst" te vermijden.

> - Laagdrempelig door modulair opbouw en duidelijk in vertalingen en uitleg.

Reviewer: ```suggestion
- Laagdrempelig door modulaire opbouw en duidelijkheid in vertalingen en uitleg.
```

Reviewer: Maar welke modules hebben we het eigenlijk over?
Bij TOOI onderscheiden we vier soorten modules:
- ontologie
- thesauri
- waardelijsten
- registers

Of bedoelen we met modules de vier use cases uit sectie 2.2? Of de bouwstenen? Daar staat: 
> Het profiel is modulair opgebouwd. Hieronder beschrijven we 4 uses cases die overeenkomen met verschillende niveaus van diepgang waarop concepten in samenhang kunnen worden beschreven. Iedere use case wordt ondersteund door een bouwsteen in het profiel.

> - We beginnen met begrippen, de overeenkomstige behoefte binnen de diverse deelnemende organisaties ligt voor een groot deel bij begrippen, dit is het laaghangende fruit voor een overeengekomen profiel.

Reviewer: ```suggestion
- Het profiel beperkt zich tot specificaties voor begrippencatalogi.
```

> - Doel van het profiel is in eerste instantie om begrippen binnen organisaties kenbaar te maken.

Reviewer: Wat willen we hiermee zeggen?
- Waarom "in eerste instantie"? Wanneer is de tweede instantie en wat gaan we dan doen?
- Het profiel is een specificatie met als doel interoperabiliteit van begrippencatalogi (BC). Een BC zou als doel kunnen hebben om begrippen kenbaar te maken, maar ons profiel toch niet?
- Waarom "binnen organisaties"? Niet erbuiten?

- We zien de Nederlanse term "begrip" als synoniem voor de Engelse term "concept".

> - Zoveel mogelijk in het Nederlands, dus zowel de labels, termen en de definities. De context is in principe de Nederlandse omgeving.

Reviewer: ```suggestion
- De context is de Nederlandse taal. De labels, termen en de definities zijn in het Nederlands.
```

> - Technisch zijn we voorbereid op de aansluiting met de internationale gemeenschap.

Reviewer: ```suggestion
- Het profiel conformeert zich zo veel mogelijk aan implementatiestandaarden die gebruikelijk zijn in de internationale gemeenschap.
```

> - Voorlopig concentreren we ons tot hiërarchische relaties en eenvoudige niet hiërarchische relaties om te kaderen. We gaan voorlopig geen domein specifieke relaties introduceren.

Reviewer: ```suggestion
- Het profiel beperkt zich tot hiërarchische relaties en eenvoudige niet hiërarchische relaties om te kaderen. Domein-specifieke relaties zijn (vooralsnog) buiten scope.
```

Reviewer: En wat zijn "eenvoudige niet hiërarchische relaties om te kaderen"?
